src/nmea.h

```diff
@@ -32,11 +32,24 @@
 #define NMEA_GGA_FIX_MANUAL  7
 #define NMEA_GGA_FIX_SIM     8
 
+#define MINUTES(X) fabs(60 * (R2D * (X) - ((s16)R2D * (X))));
```

Comment:
Check cast here too, should be:
`#define MINUTES(X) fabs(60 * (R2D * (X) - ((s16)(R2D * (X)))));`